src/test/java/org/fundacion/jala/converter/core/ExtractMetadataTest.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacion.jala.converter.core;
+
+import org.fundacion.jala.converter.core.exceptions.MetadataException;
+import org.fundacion.jala.converter.core.metadata.TypeFileExport;
+import org.junit.Test;
+
+import java.io.File;
+
+public class ExtractMetadataTest {
+
```

Comment:
I think that is not necesary this line.

This line should be removed

Reply:
it's already corrected cris